src/main/kotlin/ar/edu/unahur/obj2/vendedores/Distribucion.kt

```diff
@@ -0,0 +1,30 @@
+package ar.edu.unahur.obj2.vendedores
+
+class CentroDeDistribucion(var ciudad: Ciudad) {
+    val vendedores = mutableSetOf<Vendedor>()
+
+    fun vendedorDelCentro(unVendedor: Vendedor){
+        if(vendedores.contains(unVendedor)) {
+            throw Exception ("El vendedor ya esta trabajando en nuestro centro de distribucion")
+        }else{
+            vendedores.add(unVendedor)
+        }
```

Comment:
Si se ejecuta la rama verdadera del `if`, el método explota y nunca llega al `else`. Podés entonces obviarlo, te quedaría así:

```suggestion
        if(vendedores.contains(unVendedor)) {
            throw Exception ("El vendedor ya esta trabajando en nuestro centro de distribucion")
        }
        
        vendedores.add(unVendedor)
```